docs/manual/common/guide/devmode/CassandraServer.md

```diff
@@ -44,11 +44,11 @@ In sbt:
 
 @[cassandra-clean-on-start](code/build-cassandra-opts.sbt)
 
-# Keyspace
+## Keyspace _(deprecated)_
```

Comment:
What I'm trying to say is that I wouldn't deprecated the whole section, only the parts wrt Cassandra in DevMode (which I understand are tha parts that will be removed).